nest-forms-backend/src/nases/nases.service.ts

```diff
@@ -315,15 +305,40 @@ export default class NasesService {
       )
     }
 
+    const onlyForVerifiedUsers =
+      formDefinition.type !== FormDefinitionType.Email ||
+      formDefinition.onlyForVerifiedUsers ||
+      false
```

Comment:
`|| false` not necessary

Reply:
Without this the type would be for some reason `boolean | undefined`, so to "convert" `undefined` to `false` this is necessary.

It is `undefined` when `onlyForVerifiedUsers` is not provided, in this case we want it to be `false`.

Reply:
So by default the email form is allowed to be sent by unverified users.

Reply:
Okay, it might make sense to add it to all form definitions and make by default that the forms will be only for verified users. I will refactor it accordingly :D